### ISSUE TITLE
Base.py potentially missing

### ISSUE THREAD

**Issue by PascalStrewe** [run-llama/llama_index#10801]
### Question Validation

- [X] I have searched both the documentation and discord for an answer.

### Question

Hi team,

I was using the RecursiveRetrieverSmallToBigPack and after the migration, I get the error:

ERROR:nemoguardrails.actions.action_dispatcher:Error [Errno 2] No such file or directory: '/content/drive/MyDrive/recursive_retriever_stb_pack/llama_index/packs/recursive_retriever/base.py' while execution user_query
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/nemoguardrails/actions/action_dispatcher.py", line 178, in execute_action
    result = await result
  File "/content/drive/MyDrive/nemo-guardrails-llamaindex-rag/config/actions.py", line 56, in user_query
    print('user_message is ', user_message)
  File "/content/drive/MyDrive/nemo-guardrails-llamaindex-rag/config/actions.py", line 23, in init
    "RecursiveRetrieverSmallToBigPack", "/content/drive/MyDrive/recursive_retriever_stb_pack"
  File "/usr/local/lib/python3.10/dist-packages/llama_index/core/llama_pack/download.py", line 56, in download_llama_pack
    pack_cls = download_llama_pack_template(
  File "/usr/local/lib/python3.10/dist-packages/llama_index/core/download/pack.py", line 126, in download_llama_pack_template
    spec.loader.exec_module(module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 879, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1016, in get_code
  File "<frozen importlib._bootstrap_external>", line 1073, in get_data
FileNotFoundError: [Errno 2] No such file or directory: '/content/drive/MyDrive/recursive_retriever_stb_pack/llama_index/packs/recursive_retriever/base.py'

I (think) I download the package according to the documentation:
    RecursiveRetrieverSmallToBigPack = download_llama_pack(
        "RecursiveRetrieverSmallToBigPack", "/content/drive/MyDrive/recursive_retriever_stb_pack"
    )

The downloaded path only contains an _init_.py, no base.py.

Not sure what I am overlooking, before the migration it worked for me.

Best, 

Pascal

**Comment by nerdai** [comment]
Hi @PascalStrewe, the reason for this was actually discovered in investigating #10777. See [here](https://github.com/run-llama/llama_index/issues/10777#issuecomment-1948890434) for an explanation as to why `download_llama_pack` is not working for `RecursiveRetrieverSmallToBigPack` and 5 other llama-packs.

TL;DR: in v0.10 llama-packs are packed in python packages, where some python packages include more than one llama-pack. The `download_llama_pack` currently doesn't handle this case.

However, if you want to be able to just use `RecursiveRetrieverSmallToBigPack` and not customize it, then you can (and should) get it from pip installing the package where it comes from.

```
pip install llama-packs-recursive-retriever
```

Then to use the Pack:
```
from llama_index.packs.recursive_retriever import RecursiveRetrieverSmallToBigPack

## you should be able to use RecursiveRetrieverSmallToBigPack as before now
```

**Comment by nerdai** [comment]
(Duplicate of #10777)
